data/json/itemgroups/guns.json

```diff
@@ -200,11 +201,12 @@
       [ "ak47", 100 ],
       [ "ak74", 60 ],
       [ "an94", 40 ],
+      [ "arx160", 40 ],
+      [ "cane_rifle", 80 ],
```

Comment:
You're not going to get anywhere comparing it to other guns, you're almost
certainly right that the other guns need to be adjusted, but that doesn't
have anything to do with this gun.

To answer your question, yes, it seems extremely overpowered to me for what
it is.